File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    let words = s.split(' ');
    let answer = '';
    let answerSpace = [];
    
    words.forEach(function(word) {
        for (let i = 0; i < word.length; i++) {
            answer += i % 2 == 0 ? word[i].toUpperCase() : word[i].toLowerCase();
            
        }
        answerSpace.push(answer);
        answer = '';
    })
    return answerSpace.join(' ');
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
"""
짝수번째 알파벳은 대문자로,
홀수번째 알파벳은 소문자로 바꾼 문자열을 반환해라

단어별로 짝홀수 인덱스 판단, 첫 글자 => 짝수
"""

def solution(s):
    answer = []
    words = s.split(' ')
    
    for word in words:
        temp = ''
        for i in range(len(word)):
            if i == 0 or i % 2 == 0:
                temp += word[i].upper()
            else:
                temp += word[i].lower()
        answer.append(temp)
        
    return ' '.join(answer)